tests/test_utils.py

```diff
@@ -189,3 +190,26 @@ def test_combine_rows_sumall():
     exp = exp.reindex(columns=obs.columns)
     clean = lambda df: df.sort_index().reset_index()
     pdt.assert_frame_equal(clean(obs), clean(exp))
+
+
+def test_isin():
```

Comment:
`isin()` also is a way to filter, right? If I understand the test here, it checks that an extraneous region is not included in `exp`. Could you also add, e.g.,  

```
utils.isin(sector=["sector1"], region=["a", "b", "non-existent"])
```

to show that it also downselects?

Reply:
Downselection is already tested in parallel for both dimensions, since from the original `df`
https://github.com/iiasa/aneris/blob/491db127e101afe06cd190dd5f0d4f036438557c/tests/test_utils.py#L109-L124
we are removing rows `(sector="extra_b", region="b")` and `(sector="sector1", region="c")`.

But sure, can also add your example?

Reply:
Should I? I don't think it provides any additional insights.

Reply:
No need - I missed that this tested multiple use cases.